File: java/1832.java
import java.util.HashSet;

class Solution {
    public boolean checkIfPangram(String sentence) {

        HashSet<Character> chars = new HashSet<>();

        for(char c : sentence.toCharArray()){
            chars.add(c);

            if(chars.size() == 26){
                break;
            }
        }

        return chars.size() == 26;
    }
}

public class Main {
    public static void main(String[] args) {
        Solution s = new Solution();

        String input1 = "thequickbrownfoxjumpsoverthelazydog";
        boolean result1 = s.checkIfPangram(input1);
        System.out.println(result1);

        String input2 = "leetcode";

    }
}

File: kotlin/1832.kt
class Solution {
    fun checkIfPangram(sentence: String): Boolean {
        return sentence.toSet().size == 26
    }
}
fun main(){
    val solution = Solution()

    println(solution.checkIfPangram("thequickbrownfoxjumpsoverthelazydog"))
    println(solution.checkIfPangram("leetcode"))
}